cmd/podman/image.go

```diff
@@ -41,4 +42,8 @@ func init() {
 	_imagesSubCommand.Aliases = []string{"ls", "list"}
 	imageCommand.AddCommand(&_imagesSubCommand)
 
+	// It makes no sense to keep 'podman images rmi'; just use 'rm'
+	_rmiSubCommand.Use = strings.Replace(_rmiSubCommand.Use, "rmi", "rm", 1)
+	_rmiSubCommand.Example = strings.Replace(_rmiSubCommand.Example, "podman rmi", "podman image rm", -1)
+	imageCommand.AddCommand(&_rmiSubCommand)
```

Comment:
I asked similar question in @baude's PR, can we add a test for `podman image rm`

Reply:
So done, but wow, those ginkgo tests make my brain hurt. And they don't compile or run on my laptop. So, the test is untested. (I am bumping up the priority of my bats tests)